### PR TITLE
Introduce `segment_image` CLI

### PR DESCRIPTION
**Pull request by jcohenadad**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
- [x] I've assigned a reviewer

#### PR contents

- [x] I've consulted [ivadomed's internal developer documentation](https://github.com/ivadomed/ivadomed/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] I've added [relevant tests](https://github.com/ivadomed/ivadomed/wiki/tests) for my contribution
- [x] I've updated the [relevant documentation](https://github.com/ivadomed/ivadomed/wiki/documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description

Introduces a CLI function to segment individual images, regardless of the BIDS structure. 

## Linked issues

Addresses #1002

### CODE REVIEW

**Review by naga-karthik — COMMENTED** [review]
Looks good to me! It is an important feature to have in the CLI. I can take up the rest of the TODOs!

**Comment by jcohenadad** [comment]
> I can take up the rest of the TODOs!

Thank you @naga-karthik , I accept the offer 😊  Let's merge this and I let you open another PR for the remaining TODOs

File: ivadomed/scripts/segment_image.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        prog='segment_image',
+        description='Applies a trained model on a single image. Output are generated in the current directory.')
+    parser.add_argument("-i", "--image", nargs='+', required=True,
+                        help="Image(s) to segment. You can specify more than one image (separate with space).",
+                        metavar=imed_utils.Metavar.file)
+    parser.add_argument("-m", "--model", required=True,
+                        help="Path to folder that contains ONNX and/or PT model and ivadomed JSON config file.",
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply ivadomed/scripts/segment_image.py:25]: sorry i have no more time to dedicate to this-- pls fix it in a subsequent PR

File: ivadomed/scripts/segment_image.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+"""
+This script applies a trained model on a single image. Output are generated in the current directory.
+"""
+
+# TODO: create entry_points in setup.py and update docstrings usage
```
[the remark this answers is not in the document]
Comment (jcohenadad) [reply ivadomed/scripts/segment_image.py:6]: agreed-- sorry i have no more time to dedicate to this-- pls fix it in a subsequent PR

**Comment by jcohenadad** [comment]
@naga-karthik i let you open an issue for the remaining comments/TODOs, approve this PR and merge it pls

**Comment by naga-karthik** [comment]
Done! #1270 lists the TODOs as remaining tasks.

**Review by naga-karthik — APPROVED** [review]

**Pull request merged**